pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-mra/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, pidgin } :
+
+let
+  version = "54b2992";
+in
+stdenv.mkDerivation rec {
+  name = "pidgin-mra-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/dreadatour/pidgin-mra";
+    rev = "${version}";
+    sha256 = "1nhfx9gi5lhh2xjr9rw600bb53ly2nwiqq422vc0f297qkm1q9y0";
+  };
+
+  buildInputs = [ pidgin ];
+
+  preConfigurePhases = "preConfigurePhase";
+
+  preConfigurePhase = ''
+    sed -i 's|-I/usr/include/libpurple|$(shell pkg-config --cflags purple)|' Makefile
```

Comment:
This line is needed. The builder have access to pkg-config. I guess, pkg-config is derived from pidgin.